students/19th/hyojinkim/js/login.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const loginBtn = document.getElementsByClassName("login_btn")[0];
+const idInput = document.getElementsByClassName("login_id")[0];
+const pwInput = document.getElementsByClassName("login_pw")[0];
+const loginInfo = document.getElementsByClassName("login_info")[0];
+
+loginBtn.disabled = true;
+
+function btnActive() {
+  loginBtn.disabled = false;
+  loginBtn.style.background = "#0096F6";
+}
+function btnDisable() {
+  loginBtn.disabled = true;
+}
+
+loginInfo.addEventListener("keyup", function () {
+  if (idInput.value === "" || pwInput.value === "") {
```

Comment:
함수 전체적으로 깔끔하게 구현해주셨네요! :) 👍🏻
해당 조건문은 조건값의 boolean값에 따라 좀 더 간결하게 작성하실 수 있습니다. 
빈스트링의 boolean값은 무엇일까요? false이죠? 그러면 idInput.value값이 false일때 해당 실행문이 실행되기를 바라는 조건이겠습니다. 고로 아래와 같이 작성할 수 있겠네요! 
```suggestion
  if (!idInput.value || !pwInput.value) {
```